src/provider.rs

```diff
@@ -11,6 +11,7 @@ use rexiv2::Metadata;
 
 use crate::config::Element;
 use crate::errors::{RahmenError, RahmenResult};
+use serde::__private::Option::Some;
```

Comment:
This import seems to be out of place.

Reply:
well, no idea where it came from, and it seems to work regardless, so I'm fine with removing it.

Reply:
maybe it got added by some Intellij magick.